namedotcom/resource_namedotcom_record.go

```diff
@@ -63,6 +67,48 @@ func resourceRecordCreate(data *schema.ResourceData, meta interface{}) error {
 	return resourceRecordRead(data, meta)
 }
 
+// resourceRecordImporter import existing record from the Name.com API
+func resourceRecordImporter(data *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	//data.Id() last argument passed to terraform import, in format domain:id
+	importDomainName, importId, err := resourceRecordImporterParseId(data.Id())
+	if err != nil {
+		return nil, err
+	}
+	data.Set("domain_name", importDomainName)
+	data.SetId(importId)
+
+	// possible block to switch using format of importing id domain:host instead of domain:id
```

Comment:
This looks more user friendly, istn't it?

Reply:
I totally agree with you that "domain:host" more user friendly format, but be honest I vote for domain:id, because import feature it's not something regular for use, and for do import user must be sure of ID that he is importing, otherwise user can face issue (human error), when he has imported wrong resource, and it destroyed after on next apply step, example:
```
...
variable "mydomains"{
type = set(string)
defaults = [
  "www-dev",
  "api-dev",
]
}
resource "namedotcom_record" "records" {
  for_each = var.mydomains
  domain_name = "example.com"
  host = each.value
  record_type = "A"
  answer = "10.10.10.10"
}
...
```

lets say we need to import api, but by mistake human run command for api instead of api-dev
`terraform import 'namedotcom_record.records["api"]' example.com:api`

so in this case, "api" entry will be imported, so tfstate will know about api, but does not know about api-dev, and if user going to run `terraform apply -auto-approve`, then terraform will delete from server "API" record, because it missed in config, but exists in tfstate, and will try to create new record "API-DEV", so finally he will have "human error", 

so I would like to keep human-friendly format "domain:host", but I vote for "domain:id" because of additional type of security.

but be honest, obtain id is very simple, just one curl request, example of which can be found in readme.

final decision I leave for you